### DISCUSSION TITLE
How can I resurvey all ways for lit status now?

### DISCUSSION THREAD

**Discussion by dbdean** [streetcomplete/StreetComplete#3237] [Q&A] [answered]
Hi everyone. Now that the lit quest showing only at night times is in beta, I'd really like to resurvey every single `lit`=`yes` in my area, but I'm not sure about the best way to go about it. I want to do this, because I'm not confident that my daytime surveying looking at light poles really gave an accurate reflection of the real lit-nes of the ways.

Here are some options:

1. I remove all `lit`=`yes` from ways in my area. I'm the only mapper in my area, but it still seems a bit drastic. (I did do this back in the day, where I changed every building in town to `building`=`yes` so that I could properly survey the building type with SC)
2. I create a fork with the resurvey time on lit quests way down (say 1 year). This still won't ask on roads that have been edited for other reasons, but a lot will come up. (I'm already doing this, but the missing recently-edited ways are making it not the best solution).
3. I make a tag like `fixme:check:lit`=`yes`, add it to all lit ways in my area, and make StreetComplete respect that (at least in my fork) and remove it once the resurvey is done.
4. I create an issue to ask that StreetComplete asks questions about the [lit:perceived](https://wiki.openstreetmap.org/wiki/Key:lit:perceived) tag. This would ask on every lit=yes way what the actual quality of the light is. This would allow us to go from 'is the street lit' to 'how well is the street lit', and would also give an option to repair incorrect tagging if it turns out the way wasn't really lit at all. The tag doesn't seem super 'approved' though, although it is reasonably geographically widely used.

I'd be interested what everyone else thinks.

**Comment by Cj-Malone** [comment, marked as answer]
`check_date:lit` should trump the edit timestamp and force the resurveying of quests. So you could add a date [16 years?!?](https://github.com/streetcomplete/StreetComplete/blob/ff859e51f82e320a38ac98af9a22859692567b0d/app/src/main/java/de/westnordost/streetcomplete/quests/way_lit/AddWayLit.kt#L38) in the past.

**Reply by dbdean** [reply]
Hmm. Good point. Let me try that!

**Reply by dbdean** [reply]
Confirmed this works well.

**Comment by westnordost** [comment]
There is a plan to make a dedicated view for that - i.e. see in an overview how the streets are tagged currently and you can re-tag any of them.

**Reply by FloEdelmann** [reply]
For reference, this is #2461.
